tests/ssl/ssl.sh

```diff
@@ -1223,7 +1223,7 @@ ssl_run_tests()
                 SERVER_OPTIONS=
                 ;;
             "bypass")
```

Comment:
can we just remove the option entirely?  I mean, it will disable tests without locks on, but those should be fine, modulo any threading errors they might expose (and we're not sure that the locks will guard against that anyhow).